packaging/python/Dockerfile

```diff
@@ -33,12 +33,12 @@ RUN rpmbuild --rebuild https://vault.centos.org/8-stream/AppStream/Source/SPacka
     && yum -y install rpmbuild/RPMS/*/flex-2.6.1-9.el7.*.rpm \
     && rm -rf rpmbuild
 
-RUN wget http://ftpmirror.gnu.org/ncurses/ncurses-6.2.tar.gz \
-    && tar -xvzf ncurses-6.2.tar.gz \
-    && cd ncurses-6.2  \
-    && ./configure --prefix=/nrnwheel/ncurses --without-shared --without-debug CFLAGS="-fPIC" \
+RUN wget http://ftpmirror.gnu.org/ncurses/ncurses-5.9.tar.gz \
+    && tar -xvzf ncurses-5.9.tar.gz \
+    && cd ncurses-5.9  \
+    && ./configure --prefix=/nrnwheel/ncurses --without-shared --without-debug CFLAGS="-fPIC" CPPFLAGS="-P" --without-cxx\
```

Comment:
why are we adding the flags here?

Reply:
Otherwise won't compile with gcc10. @pramodk I don't recall the actual messages or how you determined the flags.
Note that the date for ncurses-5.9 is 2011.  I'd rather know why ncurses-6.2 does not work.  It remains a puzzle how things can go wrong when readline and ncurses are statically linked into libnrniv.so.
I'll try the latest ncurses-6.4 and see what happens. But this is a tested work around.

Reply:
Turns out that ncurses-6.4 also fixes the problem. Any downsides going with that? We can change this pr title to
"Fixes linux wheel backspace by upgrading ncurses-6.2 to 6.4" and the change does not require the extra flags.